lib/MetaCPAN/Model/Archive.pm

```diff
@@ -77,7 +77,11 @@ has _tempdir => (
     init_arg => undef,
     lazy     => 1,
     default  => sub {
-        return File::Temp->newdir;
+
+        my $scratch_disk = '/mnt/scratch_disk';
```

Comment:
Yes, agree 100%, but my role is JFDI so we get migrated to ES 2...  and indexing is taking too long.

Please do send a better PR with the data configured from metacpan_server.conf (I'll handle the metacpan_server_local.conf to match via puppet so it's specific to the server) :)